core/src/apps/cardano/sign_tx/signer.py

```diff
@@ -69,6 +73,12 @@
 _MAX_CHUNK_SIZE = const(1024)
 
 
+class SuiteTxType(IntEnum):
+    SIMPLE_SEND = 0
+    SIMPLE_STAKE = 1
+    NOT_SUITE_TX = 2
```

Comment:
Nit: maybe I missed it but I think there should be an explanation somewhere of what this enum is and how is it related to Suite.